.github/workflows/ci.yml

```
@@ -0,0 +1,90 @@
+# CI 워크플로우 이름
+name: CI
+
+# 트리거 조건: 언제 이 워크플로우가 실행될지 정의
+on:
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)